### ISSUE TITLE
tabnew crash in gvim using fastfold

### ISSUE THREAD

**Issue by zelurker** [Konfekt/FastFold#32]
This is really weird, probably a gvim bug but it seems to stay.
So :
FastFold installed, no specific configuration
open a C file from the command line with folds
type :tabnew
usually it crashes immediately, rarely it works, in this case just close the tab and try again, it crashes very fast.
Removing FastFold fixes the problem.

And the best part is that it doesn't crash when using vim (console version), only gvim crashes.
I am quite lost here... !

**Comment by Konfekt** [comment]
Ok, there's probably little FastFold can resolve here. Is this with the latest gVim? 

Does it go away after commenting

https://github.com/Konfekt/FastFold/blob/7aa18ef2003adadb12fdbeb7fc9b64ad1d4ce8f2/plugin/fastfold.vim#L169

?

Or does the option
```vim
set guioptions-=e
```
help?

**Comment by zelurker** [comment]
Yeah I found the workaround of using guioptions-=e just after posting this,
it works fine, the bug is really related to the graphic tabs in gvim then.
So for now I am using vim-flagship to handle tabs, not bad !
Sorry for the post then, but really the other way to remove this bug is to
remove Fastfold, which is extremely strange, I agree !

2016-03-09 9:08 GMT+01:00 Konfekt <notifications@github.com>:

**Comment by Konfekt** [comment]
Did it also go away after commenting

https://github.com/Konfekt/FastFold/blob/7aa18ef2003adadb12fdbeb7fc9b64ad1d4ce8f2/plugin/fastfold.vim#L169 ?

**Comment by zelurker** [comment]
Oops sorry, yes apparently it fixes the problem too.
Strange that I am the only one experiencing it...

2016-03-09 12:24 GMT+01:00 Konfekt <notifications@github.com>:

**Comment by Konfekt** [comment]
Yeah, checked over here with a gui tab line and could not reproduce.

**Comment by Konfekt** [comment]
To distill a bug report for Vim, you could try commenting out either of the lines

https://github.com/Konfekt/FastFold/blob/7aa18ef2003adadb12fdbeb7fc9b64ad1d4ce8f2/plugin/fastfold.vim#L96

and 

https://github.com/Konfekt/FastFold/blob/7aa18ef2003adadb12fdbeb7fc9b64ad1d4ce8f2/plugin/fastfold.vim#L95
